bridge/src/commonMain/kotlin/ru/herobrine1st/matrix/bridge/telegram/TelegramWorker.kt

```diff
@@ -40,50 +46,90 @@ class TelegramWorker(
             is ClientEvent.RoomEvent.MessageEvent<*> -> {
                 when (val content = event.content) {
                     is RoomMessageEventContent -> {
-                        val textContent = when (content) {
-                            is RoomMessageEventContent.TextBased.Text -> content.body
-                            else -> throw UnhandledEventException(
-                                message = "This event is not delivered due to lack of support",
-                            )
-                        }
+                        val result = when (content) {
+                            is RoomMessageEventContent.TextBased.Text -> {
+                                val textContent = content.body
+                                val bot = getBot(actorId)
```

Comment:
It should be moved higher
```suggestion
                        val bot = getBot(actorId)
                        val result = when (content) {
                            is RoomMessageEventContent.TextBased.Text -> {
                                val textContent = content.body

```